crates/miden-testing/src/kernel_tests/tx/test_input_note.rs

```diff
@@ -241,3 +241,139 @@ fn test_get_assets() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+/// Check that the number of the inputs and their commitment of a note with one asset obtained from
+/// the `input_note::get_data_info` procedure is correct.
+#[test]
+fn test_get_data_info() -> anyhow::Result<()> {
```

Comment:
We use "note inputs" and "note data" a bit inconsistenly in this PR. Ideally we make it consistent so that when we do https://github.com/0xMiden/miden-base/issues/1662, we don't have to find both "note inputs" and "note data" to rename.